project_scrum/views/project_project.xml

```diff
@@ -0,0 +1,165 @@
+                            <t t-esc="record.description.value" />
+                        </span>
+                    </h3>
+                </a>
+            </xpath>
+            <a name="%(project.act_project_project_2_project_task_all)d"
+                position="replace">
+                <a t-if="record.use_tasks.raw_value"
+                    name="%(project.act_project_project_2_project_task_all)d"
+                    type="action" style="margin-right: 10px">
+                    <span class="o_value">
+                        <t t-esc="record.task_count.raw_value" />
+                    </span>
+                    <span class="o_value">
+                        <t t-if="record.use_scrum.raw_value">Items</t>
+                    </span>
+                    <span class="o_value">
+                        <t t-if="! record.use_scrum.raw_value">Tasks</t>
+                    </span>
+                </a>
+                <a t-if="record.sprint_count.raw_value" name="%(action_ps_sprint_current)d"
+                    type="action" style="margin-right: 10px">
+                    <span class="o_value">
+                        <t t-esc="record.sprint_count.raw_value" />
+                    </span>
+                    <span class="o_label">Sprints</span>
+                </a>
+                <a t-if="record.user_story_count.raw_value" name="%(action_ps_us_current)d"
+                    type="action" style="margin-right: 10px">
+                    <span class="o_value">
+                        <t t-esc="record.user_story_count.raw_value" />
+                    </span>
+                    <span class="o_label">User Stories</span>
+                </a>
+                <a t-if="record.meeting_count.raw_value" name="%(action_ps_meeting_current)d"
+                    type="action" style="margin-right: 10px">
+                    <span class="o_value">
+                        <t t-esc="record.meeting_count.raw_value" />
+                    </span>
+                    <span class="o_label">Meetings</span>
+                </a>
+                <a t-if="record.test_case_count.raw_value" name="%(action_ps_tc_current)d"
+                    type="action" style="margin-right: 10px">
+                    <span class="o_value">
+                        <t t-esc="record.test_case_count.raw_value" />
+                    </span>
+                    <span class="o_label">Test Cases</span>
+                </a>
+            </a>
+        </field>
+    </record>
+
+    <record id="action_show_scrum_projects" model="ir.actions.act_window">
+        <field name="name">Scrum Projects</field>
+        <field name="res_model">project.project</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">kanban,form</field>
+        <field
+            name="search_view_id"
+            ref="project.view_project_project_filter" />
+        <field name="context">{'default_use_scrum': True}</field>
```

Comment:
there should be a domain on this one to actually only show scrum projects. I think this is my last nag

Reply:
@hbrunn Implemented. (not sure if you are getting notifications about my pushes)

Reply:
thank $deity not!